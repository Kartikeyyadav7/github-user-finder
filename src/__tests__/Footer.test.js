import React from "react";
import { create } from "react-test-renderer";
import Footer from "../Footer";

test("snapshot", () => {
  const c = create(<Footer />);
  expect(c.toJSON()).toMatchSnapshot();
});
