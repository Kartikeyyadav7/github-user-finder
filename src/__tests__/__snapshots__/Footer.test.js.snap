// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<footer>
  <div
    className="text-footer"
  >
    <p>
      Created by
       
      <a
        href="https://zurda.github.io/portfolio/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Michal Weizman
      </a>
      <br />
      This site code is available on
       
      <a
        href="https://github.com/zurda/github-user-finder"
        rel="noopener noreferrer"
        target="_blank"
      >
        Github
      </a>
    </p>
  </div>
</footer>
`;
